polaris-core/src/main/java/org/apache/polaris/core/persistence/PolarisEntityManager.java

```diff
@@ -51,11 +51,14 @@ public class PolarisEntityManager {
   /**
    * @param metaStoreManager the metastore manager for the current realm
    * @param credentialCache the storage credential cache for the current realm
+   * @param entityCache the entity cache
    */
   public PolarisEntityManager(
-      PolarisMetaStoreManager metaStoreManager, StorageCredentialCache credentialCache) {
+      PolarisMetaStoreManager metaStoreManager,
```

Comment:
Feels like `PolarisEntityManager` should become a CDI bean as well and get the instances injected.

Reply:
That would effectively eliminate the need for `RealmEntityManagerFactory` - which is fine, but it means the callers need to be made either realm-scoped or request-scoped

Reply:
Callers should be request-scoped, no?

Reply:
I asked myself this question as well when doing the Quarkus port. But in the end I left `PolarisEntityManager` as a non-CDI class. There might be room for improvement around here.